frontend/public/components/cluster-service-plan.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable no-undef */
+
+import * as React from 'react';
```

Comment:
Add a space between this import and the console imports

Reply:
fixed